.github/workflows/coverity.yaml

```diff
@@ -0,0 +1,70 @@
+name: Coverity
+on:
+  schedule:
+    # run on every saturday
+    - cron: '0 0 * * SAT'
```

Comment:
Can it be better to run it after the usual regression test? Now it runs 20 mins before other regression tests, right?

Reply:
it runs 4 hours after, the cron syntax is
minute hour day month weekday

but i will put at 22 so its 2hours after main run